.github/workflows/ci.yml

```diff
@@ -12,14 +12,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust_version: [1.36.0, stable, nightly]
+        rust: [ 1.36.0, stable ]
+        include:
+          # Test nightly but don't fail the build.
+          - rust: nightly
+            experimental: true
```

Comment:
By the way, `experimental` by itself doesn't do or mean anything; you need to set `continue-on-error` based on the value of `experimental` (see the wg template CI for an example)

Reply:
Thank you, fixed!